Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -188,34 +212,65 @@ public static string GetJsonReportPath(string jsonPath)
 			return string.Empty;
 		}
 
-		public static void CreateDirectory(string path)
+		public void CreateDirectory(string path)
 		{
 			if (!Directory.Exists(path))
```

Comment:
Same as before, you don't have to check the existence of the Directory

Reply:
I've removed the redundant code in all cases where the Directory was created